src/pages/OnboardingPage/index.js

```diff
@@ -206,12 +211,12 @@ export default class Onboarding extends Component {
 					</ul>
 				</div>
 				<div className="right">
-					<a className="skip-link" href="/">
+					<button type="button" className="skip-link" onClick={this.skipTutorial}>
```

Comment:
You don't need `type="button"` with a `button`.

Reply:
Also, this has affected the design:

![image](https://user-images.githubusercontent.com/9116042/46065062-bc17e700-c18e-11e8-91c3-843492ae6bef.png)

Reply:
If we remove `type="button"` it gives linting error